ui/raidboss/popup-text.js

```diff
@@ -324,6 +374,12 @@ class PopupText {
           }
         }
       };
+      let resolveText = (condition, callback, triggerId) => {
```

Comment:
Can you add some documentation in https://github.com/quisquous/cactbot/blob/master/ui/raidboss/data/triggers/README.txt for how this new feature works?